dir.props

```diff
@@ -485,6 +485,11 @@
       </PropertyGroup>
     </Otherwise>
   </Choose>
+  
+  <!-- Default Test platform to deploy the netstandard compiled tests to -->
+  <PropertyGroup>
+    <TestTFM Condition="'$(TestTFM)'==''>netcoreapp1.0</TestTFM>
```

Comment:
I just want to make sure when building locally or through the CI, we'll run the tests as netcorapp1.0.   

---
In reply to: [65790351](https://github.com/dotnet/corefx/pull/9129#discussion_r65790351) [](ancestors = 65790351)